src/main/java/io/xpring/xrpl/RawTransactionStatus.java

```diff
@@ -11,14 +12,25 @@
     /**
      * Create a new RawTransactionStatus from a {@link TransactionStatus} protocol buffer.
      *
-     * @param transactionStatus The transaction status to encapsulate.
+     * @param transactionStatus The {@link TransactionStatus} to encapsulate.
      */
     public RawTransactionStatus(TransactionStatus transactionStatus) {
         this.validated = transactionStatus.getValidated();
         this.transactionStatusCode = transactionStatus.getTransactionStatusCode();
         this.lastLedgerSequence = transactionStatus.getLastLedgerSequence();
     }
 
+    /**
+     * Create a new RawTransactionStatus from a {@link GetTxResponse} protocol buffer.
+     *
+     * @param getTxResponse The {@link GetTxResponse} to encapsulate.
+     */
+    public RawTransactionStatus(GetTxResponse getTxResponse) {
```

Comment:
Is it normal to overload method signatures like this in Java?

I see `public RawTransactionStatus(...)` right above this.

Reply:
I think that this is fine. 

The idea here is that this is a bridge class that conforms both of these inputs (from the new and old protocol buffers) to a single type so that `XpringClientDecorator` is fully compatible. 

I'm going to defer to the local java experts.

Reply:
Perfectly reasonable to have two implementations of a constructor (or method) that accept different arguments. Happens quite often in open source libraries.